### PR TITLE
include json packages

### PR DESCRIPTION
**Pull request by robbecker-wf**
### Summary

Include a straight copy of json_annotation at version 3.1.1, but renamed to json_annotation_3_1_1.

Include a straight copy of json_serializable at version 3.5.2, but with minor needed changes for Workiva
- renamed to json_serializable_3_5_1
- changed the dependency on oss json_annotation to the internal published one

Testing
- ensure that the 2 new artifacts are included in the build artifacts and that they are the right packages and names.

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: json_annotation-3.1.1/pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: json_annotation_3_1_1
```
Comment (robbecker-wf) [inline json_annotation-3.1.1/pubspec.yaml:1]: renamed the package. This is the only change to the json_annotation package.

File: json_serializable-3.5.2/pubspec.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+name: json_serializable_3_5_2
```
Comment (robbecker-wf) [inline json_serializable-3.5.2/pubspec.yaml:1]: renamed the package

File: json_serializable-3.5.2/pubspec.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+  # Use a tight version constraint to ensure that a constraint on
+  # `json_annotation` properly constrains all features it provides.
+  # For v3 only – allow a wide range since the only change was to REMOVE things
+  # from json_annotation
+  json_annotation_3_1_1:
+    hosted:
+      name: json_annotation_3_1_1
+      url: https://pub.workiva.org
+    version: ^5.3.0 # This version doesn't matter, there's only the 3_1_1 forked version
```
Comment (robbecker-wf) [inline json_serializable-3.5.2/pubspec.yaml:23]: changed the json_annotation dependency to the temporary copied name

File: Dockerfile [hunk 4]
```diff
@@ -27,6 +27,12 @@ WORKDIR /build/dio
...
 RUN create_publishable_artifact.sh
 
-ARG BUILD_ARTIFACTS_PUB=/build/dio/pub_package.pub.tgz
+WORKDIR /build/json_annotation-3.1.1
+RUN create_publishable_artifact.sh
+
+WORKDIR /build/json_serializable-3.5.2
+RUN create_publishable_artifact.sh
+
+ARG BUILD_ARTIFACTS_PUB=/build/dio/pub_package.pub.tgz:/build/json_annotation-3.1.1/pub_package.pub.tgz:/build/json_serializable-3.5.2/pub_package.pub.tgz
```
Comment (robbecker-wf) [inline Dockerfile:36]: Add the new packages as build artifacts

**Review by dustinlessard-wf — APPROVED** [review]

**Comment by robbecker-wf** [comment]
QA+1 CI passes
@Workiva/release-management-p let's publish em!

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**